### PR TITLE
update infer lib to 2.0.0

### PR DESCRIPTION
**Pull request by cryoco**
update infer lib to 2.0.0

### CODE REVIEW

File: doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_en.rst [hunk 1]
```diff
@@ -53,8 +54,8 @@ Firstly we pull the latest code from github.
 
   git clone https://github.com/paddlepaddle/Paddle
   cd Paddle
-  # Use git checkout to switch to stable versions such as v1.8.5
-  git checkout v1.8.5
+  # Use git checkout to switch to stable versions such as v2.0.0
+  git checkout v2.0.0
```
Comment (swtkiwi) [inline doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_en.rst]: git checkout release/2.0

File: doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_cn.rst [hunk 2]
```diff
@@ -49,7 +53,7 @@ WITH_TENSORRT                 OFF            编译支持NVIDIA TensorRT的预
   git clone https://github.com/paddlepaddle/Paddle
   cd Paddle
   # 建议使用git checkout切换到Paddle稳定的版本，如：
-  git checkout v1.8.5
+  git checkout v2.0.0
```
Comment (swtkiwi) [inline doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_cn.rst]: git checkout release/2.0

File: doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_en.rst (hunk 1 above)
Comment (cryoco) [reply doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_en.rst]: done

File: doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_cn.rst (hunk 2 above)
Comment (cryoco) [reply doc/paddle/guides/05_inference_deployment/inference/build_and_install_lib_cn.rst]: done

**Review by swtkiwi — APPROVED** [review]
LGTM

**Pull request merged**